lib/membership/member.js

```diff
@@ -173,7 +173,7 @@ Member.prototype._applyUpdatePenalty = function _applyUpdatePenalty() {
 Member.prototype._isLocalOverride = function _isLocalOverride(update) {
```

Comment:
@Motiejus the comment in #261 on `_isLocalOverride` is not. If `_isLocalOverride` returns true, the update will be changed to assert aliveness (and bump incarnationNumber); if not and it's also not an other overwrite, it's skipped.